lib/jekyll/site.rb

```diff
@@ -158,7 +164,6 @@ def render
         end
       end
 
-      payload = site_payload
```

Comment:
This line is non-negotiable. Totally changes the speed of regen. For one site, it went from 300 seconds to 80 seconds as a result of this line. I'd like to keep it at all costs or cache the site payload on first call.